src/lsm/superblock_free_set.zig

```diff
@@ -72,17 +72,30 @@ pub const SuperBlockFreeSet = struct {
 
     /// Returns the number of released blocks.
     /// Excludes blocks staged to be released.
-    pub fn count_released(set: *SuperBlockFreeSet) u64 {
+    pub fn count_released(set: SuperBlockFreeSet) u64 {
         return set.blocks.count();
     }
 
     /// Returns the number of acquired blocks.
     /// Includes blocks staged to be released.
-    pub fn count_acquired(set: *SuperBlockFreeSet) u64 {
+    pub fn count_acquired(set: SuperBlockFreeSet) u64 {
         return set.blocks.capacity() - set.blocks.count();
     }
 
-    // TODO Add unit tests for count_released() and count_acquired().
+    /// Returns the address of the highest allocated block.
+    pub fn last_acquired(set: SuperBlockFreeSet) ?u64 {
```

Comment:
I think this function name could be improved. If I understand correctly, the highest allocated block isn't necessarily the most recently acquired block as a block with a lower address may have been free'd and re-acquired since the highest block was acquired.

I realize now after writing this that you probably meant "last" as in "physically last in order on disk" instead of "most recently acquired" as I read it, I still think it would be worth resolving this ambiguity though.

Reply:
That is a good point — the idea is to provide a (very rough) estimate of the amount of disk used, so the relevant "last" block has the highest address, but is not necessarily last chronologically.

Reply:
@sentientwaffle to borrow from your comment, what about `highest_allocated_block_address()`?